lib/options/option_validator.rb

```diff
@@ -32,7 +32,18 @@ def validate(options)
 
     # Validates BrowserStack options
     def validate_bs(options, errors)
-      errors << "--#{Maze::Option::APP} must be specified" if options[Maze::Option::APP].nil?
+      # App
+      app = options[Maze::Option::APP]
+      errors << "--#{Maze::Option::APP} must be specified" if app.nil?
+      unless app.start_with? 'bs://'
+        errors << "app file '#{app}' not found" unless File.exist? app
+      end
```

Comment:
Yeah, that's a bit heinous.  As it turns out, though, this file has had a bit of a rework on the next PR to come (which absorbs the driver and options for Browser testing into MazeRunner). I'll leave as is, as it will only conflict and require dropping due to the nature of the coming changes.